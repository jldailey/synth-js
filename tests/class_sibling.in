foo.bar
	goo.baz
		hoo.bap
		a
