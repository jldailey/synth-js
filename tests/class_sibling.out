<foo class="bar">
  <goo class="baz">
    <hoo class="bap"/>
    <a/>
  </goo>
</foo>
